sdk/reservations/arm-reservations/CHANGELOG.md

```diff
@@ -1,15 +1,18 @@
 # Release History
+    
+## 9.0.0 (2023-03-08)
+    
+**Features**
 
-## 8.0.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+  - Interface CatalogMsrp has a new optional parameter p3Y
+  - Interface CatalogMsrp has a new optional parameter p5Y
 
-### Other Changes
+**Breaking Changes**
 
+  - Delete parameters of ReturnPostHeaders in TypeAlias ReturnPostResponse
+  - Delete parameters of RefundResponse in TypeAlias ReturnPostResponse
```

Comment:
This breaking is expected from pr https://github.com/Azure/azure-rest-api-specs/pull/22622/files#diff-4bfcd2446130dd563053c9923f4c16df26d0f3098cf1480cabb180e7101ac1ac

Reply:
I am good with this, seems there is no way we could recover from that.